.flake8

```diff
@@ -1,8 +1,63 @@
 [flake8]
 
 max-line-length = 119
-ignore = E203, E501, E701, E704, W503
+ignore =
+    # black disagrees with flake8 about these
+    E203, E501, E701, E704, W503
 
+    # Assigning to `os.environ` doesn't clear the environment.
+    B003
```

Comment:
This looks valid, I didn't check how do we use os.environ, maybe we should pass it to subprocesses explicitly as it suggests.